Gems/Atom/RPI/Code/Source/RPI.Public/Pass/Specific/DownsampleSinglePassMipChainPass.cpp

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Atom/RHI/Factory.h>
+#include <Atom/RPI.Public/Pass/Specific/DownsampleSinglePassMipChainPass.h>
+#include <Atom/RPI.Reflect/Buffer/BufferAssetCreator.h>
+
+// For "A_CPU" macro, refer ffx_a.h and below:
+// https://github.com/GPUOpen-Effects/FidelityFX-SPD/blob/c52944f547884774a1b33066f740e6bf89f927f5/sample/src/DX12/SPDIntegration.hlsl
+#define A_CPU
+#ifdef _MSC_VER
+#pragma warning(disable: 4505) // unreferenced local function has been removed.
+#endif
+#include <ffx_a.h>
+#include <ffx_spd.h>
+
+namespace AZ
```

Comment:
you can write this as `namespace AZ::RPI` to remove a level of indentation

Reply:
Thanks.  I thought such a style was prohibitted since I have not seen it, and I just saw it in the Coding Standard.  I fixed both .h and .cpp.